itou/openid_connect/models.py

```diff
@@ -11,6 +11,12 @@
 from .constants import OIDC_STATE_EXPIRATION
 
 
+class TooManyKind(Exception):
```

Comment:
Je suggère 
```suggestion
class TooManyKindsException(Exception):
```

Kind au pluriel car l'exception est levée s'il y en a plusieurs et Exception pour être explicite sur la classe.